### PR TITLE
Resolves #156038187 - Create Modal component

### PR DESCRIPTION
**Pull request by john555**
#### Description
Add Modal component. When the FAB is clicked, the modal is visible.
#### Screenshot.
screen shot 2018-03-20 at 11 12 22 am

#### How to test.
- Clone this branch and checkout to it.
- Sign in to the app and click the FAB at the bottom right. A modal should appear.

### CODE REVIEW

File: src/containers/App.jsx [hunk 1]
```diff
@@ -40,6 +43,33 @@ class App extends Component {
     }
   }
 
+  onFabClick = (event) => {
+    if (event.type === 'keydown' && event.keyCode !== 13) {
```
Comment (mkiterian) [inline src/containers/App.jsx]: What does this do?

Comment (john555) [reply src/containers/App.jsx]: It makes the modal to be opened by hitting the ENTER key when focus is on the FAB.

Comment (Farhanisto) [reply src/containers/App.jsx]: Seems not to be working at the moment

Comment (john555) [reply src/containers/App.jsx]: How did you test it @Farhanisto ?

Comment (Farhanisto) [reply src/containers/App.jsx]: I fetched your branch and ran it... when the mouse  is on the component and i press enter the model is not revealed.

Comment (john555) [reply src/containers/App.jsx]: @Farhanisto use the Tab key to navigate a component then press enter.

Comment (Farhanisto) [reply src/containers/App.jsx]: Cool it works

Comment (mkiterian) [reply src/containers/App.jsx]: 👍

**Pull request merged**